README.md

```diff
@@ -88,10 +88,18 @@ The `docker-compose.yml` file must be edited to match your needs.
 Set the port and volume (equivalent to `-p` and `-v` arguments):
```

Comment:
We could make `docker-compose.yml` word in the previous line (l87) a link to the [existing docker compose file example](https://github.com/bastienwirtz/homer/blob/main/docker-compose.yml).